src/test/compressor/test_compression.cc

```diff
@@ -0,0 +1,153 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
Test cases you removed were Intended for plugin load verification only. There is a couple more named test_compression_snappy (_zlib) that verify compression/decompression functionality. IMHO worth combining into a single file too.